.travis.yml

```diff
@@ -68,7 +69,7 @@ env:
 
 before_install:
   - unset SBT_OPTS JVM_OPTS
-  - docker pull ziemin/pio-testing
+  - docker pull chanlee514/pio-testing
```

Comment:
That sounds good. Once this is done, I think we should add documentation on how to experiment/contribute to the Docker image as well.